omero/sysadmins/limitations.txt

```diff
@@ -143,6 +143,15 @@ clients will use the extrema of the pixel type range by default. Users can
 adjust the minima/maxima via the rendering settings. Recalculating minima and
 maxima pixel values after import is currently not supported.
 
+Flex data with Acapella results in OMERO.tables
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are currently issues with the configuration required to access the raw
```

Comment:
Perhaps something along the lines of "Anyone using the advanced configuration setting `FlexReaderServerMaps` for importing Flex data split between multiple directories should not upgrade to 5.1.0"?